testsuite/tests/apicast/policy/test_liquid.py

```diff
@@ -0,0 +1,67 @@
+"""
+Test behavior of liquid
+"""
+
+from packaging.version import Version  # noqa # pylint: disable=unused-import
+
+import pytest
+
+from testsuite import rawobj
+from testsuite import TESTED_VERSION  # noqa # pylint: disable=unused-import
+
+
+pytestmark = [
+    pytest.mark.skipif("TESTED_VERSION <= Version('2.12')"),
+    pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-8483"),
```

Comment:
Wouldn't it be better for the marks to be on the appropriate tests instead of on the entire module?

Reply:
Entire module is appropriate (so far)